sim_instr/src/sim_instr_node

```diff
@@ -0,0 +1,50 @@
+#include "ros/ros.h"
+#include "geometry_msgs/Twist.h"
+//#include <node_handle.h>
+#include <vector>
+#include <string>
+#include <fstream>
+#include <sstream>
+
+struct ParsedLine{
+    double time;
+    geometry_msgs::Twist twist;
+};
+
+ParsedLine parse_line(const std::string &line_in) {
+    std::istringstream line_stream(line_in);
+    ParsedLine line;
+    line_stream >> line.time >> line.twist.linear.x >> line.twist.linear.y >> line.twist.linear.z >>
+        line.twist.angular.x >> line.twist.angular.y >> line.twist.angular.z;
+    return line;
+}
+
+void run(int argc, char** argv) {
+    ros::init(argc, argv, "sim_instr");
+    ros::NodeHandle nh;
+    ros::Publisher pub = nh.advertise<geometry_msgs::Twist>("cmd_vel", 1000);
+
+    std::string filename;
+    nh.getParam("filename", filename);
```

Comment:
check this return value and throw an exception if false to avoid using an uninitialized/garbage value